_tests/FooterMenu.test.js

```diff
@@ -0,0 +1,25 @@
+import env from './_env';
+
+jest.setTimeout(env.timeout);
+
+
+describe('FooterLinks', () => {
+  beforeAll(async () => {
+    await page.goto(`${env.baseUrl}`);
+  });
+
+  it('should contain specific links in the Footer menu', async () => {
+    const footerMenuItems = [
+      'Privacy',
+      'Terms of use'
+    ];
+    const footerMenuItemsHref = [
+      'https://www.un.org/en/sections/about-website/privacy-notice',
+      'https://www.un.org/en/sections/about-website/terms-use'
```

Comment:
```suggestion
      'https://www.un.org/en/sections/about-website/privacy-notice/',
      'https://www.un.org/en/sections/about-website/terms-use/'
```

This test was failing until I added trailing slashes:

![CD-193-FooterMenu](https://user-images.githubusercontent.com/254753/113865927-fc6b6d80-97ac-11eb-9cf9-918ef7e503e5.png)

Reply:
Whoops, meant to include that it was the Feature site where the test failed.

Reply:
@rupl I'm gonna opt to change the URLs in the footer on the Feature site, rather than change the test. I assume the URL with the trailing slash was a copy-and-paste error and not intentional or necessary.
Also making a note to myself here as I noticed the UN site has redirects in place and the URLs could be https://www.un.org/en/about-us/privacy-notice and https://www.un.org/en/about-us/terms-of-use instead.